stream_alert_cli/package.py

```diff
@@ -24,26 +24,27 @@
 import tempfile
 import pip
 
-from stream_alert_cli.helpers import CLIHelpers
-
 import boto3
 
+# from stream_alert_cli.logger import LOGGER_CLI
+logging.basicConfig(format='%(name)s [%(levelname)s]: %(message)s')
+LOGGER_CLI = logging.getLogger('StreamAlertCLI')
+LOGGER_CLI.setLevel(logging.INFO)
+
 class LambdaPackage(object):
     """Build and upload a StreamAlert deployment package to S3."""
-    package_folders = ()
-    package_files = ()
+    package_folders = {}
+    package_files = {}
```

Comment:
Here these are initialized as `dicts`, but the subclasses define them as `sets`. So this should be
```Python
package_folders = set()
package_files = set()
```

However, I think a better way to do this is with [abstract properties](https://docs.python.org/2/library/abc.html#abc.abstractproperty):
```Python
class LambdaPackage(object):
  __metaclass__ = abc.ABCMeta

  @abc.abstractproperty
  def package_folders(self):
    """A set of string folder names..."""
    pass

  @abc.abstractproperty
  def package_files(self):
    """A set of string package file names."""
    pass


class RuleProcessorPackage(LambdaPackage):  
  @property
  def package_folders(self):
    return {'stream_alert/rule_processor', 'rules', 'conf'}
  
  @property
  def package_files(self):
    return {'stream_alert/__init__.py', 'variables.json'}
```

It's annoyingly verbose, but it guarantees that all of the fields are defined by all subclasses and makes the code more resilient to future changes. The properties are accessed just like they are right now (`RuleProcessorPackage().package_folders`)